UOP1_Project/Assets/Scripts/FSMAction.cs

```diff
@@ -0,0 +1,45 @@
+﻿namespace UOP1.FSM35
+{
+	public delegate void StateAction();
+
+	class ActionKey
+	{
+		public int SourceStateID;
+		public int SourceTransitionID;
+	}
+
+	public class FSMAction<TState, TTransition>
+	{
+		private FSMState<TState> initialState;
+		private FSMTransition<TTransition> transition;
```

Comment:
I'm seeing tabs in some files and spaces in others. Is there a codestyle that's been decided on?

Reply:
Not yet, the code-style is still in the works. It will be posted on the forum and somewhere in Github as well most likely once we have one :)

Reply:
@MileyHollenberg even if it was not established yet it should at least be the same throughout the files for now. Wouldn't you agree?

Reply:
Once we have the code-style in place we'll also make sure to add an automatic linter so that we don't have to worry too much about these specifics :) (will be tabs instead of spaces btw). For now I don't see this as a priority for this PR